src/components/Panels/ElementsPanel/ElementsPanel.vue

```diff
@@ -0,0 +1,186 @@
+                    :key="category[0]"
+                >
+                    <v-expansion-panel-title>
+                        {{
+                            t(
+                                'simulator.panel_body.circuit_elements.expansion_panel_title.' +
+                                    category[0]
+                            )
+                        }}
+                    </v-expansion-panel-title>
+                    <v-expansion-panel-text>
+                        <div class="panel customScroll">
+                            <div
+                                v-for="element in category[1]"
+                                :id="element.name"
+                                :key="element"
+                                :title="element.label"
+                                class="icon logixModules"
+                                @click="createElement(element.name)"
+                                @mouseover="getTooltipText(element.name)"
+                                @mouseleave="tooltipText = 'null'"
+                            >
+                                <img
+                                    :src="element.imgURL"
+                                    :alt="element.name"
+                                />
+                            </div>
+                        </div>
+                    </v-expansion-panel-text>
+                </v-expansion-panel>
+            </v-expansion-panels>
+            <div
+                id="Help"
+                lines="one"
+                :class="tooltipText != 'null' ? 'show' : ''"
+            >
+                {{ tooltipText }}
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import PanelHeader from '@/Panels/PanelHeader.vue'
+import metadata from '#/simulator/src/metadata.json'
+import simulationArea from '#/simulator/src/simulationArea'
+import { uxvar } from '#/simulator/src/ux'
+import modules from '#/simulator/src/modules'
+import { onBeforeMount, ref } from 'vue'
+var panelData = []
+window.elementHierarchy = metadata.elementHierarchy
+window.elementPanelList = []
+
+import { useI18n } from 'vue-i18n'
+const { t, locale } = useI18n()
+
+onBeforeMount(() => {
+    for (const category in elementHierarchy) {
+        var categoryData = []
+        for (let i = 0; i < elementHierarchy[category].length; i++) {
+            const element = elementHierarchy[category][i]
+            if (!element.name.startsWith('verilog')) {
+                window.elementPanelList.push(element.label)
+                element['imgURL'] = getImgUrl(element.name)
+                categoryData.push(element)
+            }
+        }
+        panelData.push([category, categoryData])
+    }
+})
+
+function getImgUrl(elementName) {
+    const elementImg = new URL(
+        `../../../assets/img/${elementName}.svg`,
+        import.meta.url
+    ).href
+    return elementImg
+}
+
+var elementInput = ref('')
+function searchElements() {
```

Comment:
Please make the element search case-insensitive. Currently, I can I'm not able to search for Hex using text "Hex", only "hex" is working.

Reply:
Fixed it.